### PR TITLE
Release notes 1st August

### PR DESCRIPTION
**Pull request by aapeliv**
+ other tweaks

**Frontend checklist**
- [ ] Formatted my code with `yarn format && yarn lint --fix`
- [ ] There are no warnings from `yarn lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/frontend/src/features/dashboard/constants.ts
Comment (ItsiW) [inline app/frontend/src/features/dashboard/constants.ts]: add a link

File: app/frontend/src/features/dashboard/constants.ts
Comment (ItsiW) [inline app/frontend/src/features/dashboard/constants.ts]: *contributor form

**Comment by aapeliv** [comment]
Waiting for a bit more before merging, but ready for review of the frontend bits.

**Review by darrenvong — DISMISSED** [review]
Frontend looks good to me!

**Pull request merged**
